src/effects/Wahwah.h

```diff
@@ -37,11 +37,42 @@ class EffectWahwahState
    double b0, b1, b2, a0, a1, a2;
 };
 
-class EffectWahwah final : public StatefulPerTrackEffect
+
+struct EffectWahwahSettings
+{
+    /* Parameters:
+    mFreq - LFO frequency
+    mPhase - LFO startphase in RADIANS - useful for stereo WahWah
+    mDepth - Wah depth
+    mRes - Resonance
+    mFreqOfs - Wah frequency offset
+    mOutGain - output gain
+
+      !!!!!!!!!!!!! IMPORTANT!!!!!!!!! :
+      mDepth and mFreqOfs should be from 0(min) to 1(max) !
+      mRes should be greater than 0 !  */
+
+   static constexpr double mFreqDefault = 1.5;
```

Comment:
We aren't strict about naming conventions.  But the "m" prefix suggests a nonstatic data member, while these are just constants.  I would simply use no prefix.